### PR TITLE
Update message shown when invalid index for pool is given

### LINKED ISSUE #148: [PE-D] Misleading error message for `unpool INVALID_INDEX`
When the INDEX provided for `unpool` command is larger than the number of pools in the pool list, the error message provided is `One of the passenger indexes provided is invalid` as shown in the image below.

This error message is misleading as it is not the passenger index provided that is invalid but the pool index that is invalid. As seen, this error message is used correctly elsewhere when the pool includes invalid index for the passengers in pool.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Ellevy/ped#1

### PR DESCRIPTION
**Pull request by chewterence**

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -9,8 +9,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PASSENGER_DISPLAYED_INDEX = "One of the passenger index"
             + " provided is invalid";
-    public static final String MESSAGE_INVALID_POOL_DISPLAYED_INDEX = "One of the pool index"
-            + " provided is invalid";
+    public static final String MESSAGE_INVALID_POOL_DISPLAYED_INDEX = "The pool index provided is invalid";
```
Comment (lyueyang) [inline src/main/java/seedu/address/commons/core/Messages.java:12]: Just wondering, what's the rationale for the change considering that the old message would be in line with the previous message for passengers instead

Comment (chewterence) [reply src/main/java/seedu/address/commons/core/Messages.java:12]: was addressing issue #148 but i realised that for deleting passengers, multiple indexes can be provided and thus "one of" would be suitable. However, for unpool its only one index, hence the "One".

**Comment by lyueyang** [comment]
resolves #148

**Review by ZechariahTan — APPROVED** [review]
LGTM
